From d72e21d43c9ff06181c78d059ba03fe92d1accdb Mon Sep 17 00:00:00 2001
From: Sultanxda <sultanxda@gmail.com>
Date: Wed, 9 Sep 2015 18:23:59 -0700
Subject: [PATCH 1/3] liblog: Silence spammy logs from camera blobs (AEC_PORT
 and mm-camera)

Log messages under these tags are spammed very hard, destroying logs, and
the messages that are spammed are not useful for debugging.

Change-Id: I8017c2ae1c04f1f08b8027eeb4f7a70249c72343
Signed-off-by: Kshitij Gupta <kshitijgm@gmail.com>
---
 liblog/logprint.c | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/liblog/logprint.c b/liblog/logprint.c
index b62f8b446..9e3f9ea71 100644
--- a/liblog/logprint.c
+++ b/liblog/logprint.c
@@ -201,7 +201,10 @@ static android_LogPriority filterPriForTag(AndroidLogFormat* p_format,
 LIBLOG_ABI_PUBLIC int android_log_shouldPrintLine(AndroidLogFormat* p_format,
                                                   const char* tag,
                                                   android_LogPriority pri) {
-  return pri >= filterPriForTag(p_format, tag);
+    if (!strncmp(tag, "AEC_PORT", 8) || !strncmp(tag, "mm-camera", 9))
+        return 0;
+    else
+        return pri >= filterPriForTag(p_format, tag);
 }
 
 LIBLOG_ABI_PUBLIC AndroidLogFormat* android_log_format_new() {
-- 
2.17.1

